### DISCUSSION TITLE
请教 研究了一晚上还是没明白custom.yaml怎么配置

### DISCUSSION THREAD

**Discussion by Daohuai0913** [kkkgo/PaoPaoGateWay#109] [问和答]
只有一个global分组 又不支持增加proxies-groups 这样怎么分流呢？rules里面分别用 global 和direct手动分流？

**Comment by Daohuai0913** [comment]
我把网关设置成了ppGateway 这样就全部走代理 该怎么设置呢 没明白

**Comment by jaesony** [comment]
本来就是全局代理网关，用paopaodns或者别的DNS分流
